core/src/duplicate_shred.rs

```diff
@@ -114,24 +105,65 @@ fn check_shreds(
 pub fn from_duplicate_slot_proof(
```

Comment:
Is this function used anywhere now that we have `from_shred()`?

Reply:
no. I am going to delete it once this code is finalized.